### ISSUE TITLE
DM EELS spectra do not read exposure from original_metadata

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#1859]
I'm observing that for EELS spectra imported from dm3 files, the exposure (or dwell time or "integration time" depending on names) is not correctly placed in the metadata.

This is valid for dm3 files recorded in Oxford and Oslo on separate instruments.

I noticed this in connection with #1851 as my lowloss and coreloss had not been acquired with similar exposure times. I then realised that it appears that the eelsmodel convolved with lowloss does not take into account exposure times or the number of frames, even if they are provided.

**Comment by francisco-dlp** [comment]
It'll be useful to share a file that suffers from the issue.

Regarding this information not being used automatically, I agree that it would be a nice feature to add. There are several reasons why it doesn't work like this right now:

1. At the time of writing the code it was not clear to me if the exposure information in DM was accurate enough. For example, I didn't know if always (no matter the acquisition options) the exposure stored is the real exposure. Now I think that it is, but I am not fully sure.
2. Sometimes the dwell time is wrong because the file has been post-processed with external packages (e.g. binning) and the effective dwell time has not been updated in the file.
3. Because of 1., implementing this now will break the people's scripts.

My thinking was that, by not taking into account the dwell time for the computation the result would be obviously wrong to anybody interested in absolute quantification (few at the time, many today) who would take the necessary steps to fix it knowing what they were doing. In hindsight, I think that this was a too cautious attitude that has led to a more severe problem (3.). As this has to be fixed as some point, I now think that the sooner the better.

**Comment by thomasaarholt** [comment]
I've attached the smallest file I could find. 
[EELS Spectrum Image.dm3.zip](https://github.com/hyperspy/hyperspy/files/1794554/EELS.Spectrum.Image.dm3.zip)

I'm fixing this now, but I'm unsure if I should go ahead and overwrite the (for my files) non-working code, or introduce a new line. I don't fully appreciate where this one is coming from? EFTEM? Old detectors?

Changing [line 918 in `digital_micrograph.py`](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/io_plugins/digital_micrograph.py#L918):
```python
"{}.EELS.Acquisition.Integration time (s)".format(tags_path): (
                    "Acquisition_instrument.TEM.Detector.EELS.%s" % mapped_attribute,
                    None),
```
to:
```python
"{}.EELS.Acquisition.Exposure_s".format(tags_path): (
                    "Acquisition_instrument.TEM.Detector.EELS.%s" % mapped_attribute,
                    None),
```
introduces the following parameter to the metadata for the file I attached above:
`├── dwell_time = 0.01`

**Comment by thomasaarholt** [comment]
@ericpre `git blame`s you for the reading of the Integration time parameter. Do you remember why you added it?

**Comment by francisco-dlp** [comment]
Fixes in #1901
